packages/app/components/Toast/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Pressable, StyleProp, ViewStyle } from "react-native";
+import styled, { css } from "styled-components/native";
+import { hexToRGB } from "../../../../apps/next/src/helpers/misc";
```

Comment:
![image](https://user-images.githubusercontent.com/102537764/172585577-69c2a5e4-620e-4d93-b901-6bdd2d5d4319.png)

please, create helpers folder inside packages/app and copy in this folders misc file